src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -59,6 +59,11 @@ protected override Assembly Load(AssemblyName assemblyName)
         Console.WriteLine("CustomAssemblyLoader: Got request to load {0}", assemblyName.ToString());
 
         string strPath;
+        if (assemblyName.Name.StartsWith("System.Console"))
```

Comment:
It is strange to load all system files from one location and use the `System.Console` from the default context. It would likely result in using other assemblies (direct dependencies of `System.Console`) from the default context.
It seems we should rather call the `Console.SetOut(System.IO.TextWriter.Null);` on the Console from the loaded copy of the `System.Console` (using reflection) when the `SuppressConsoleOutputFromTests` is set. I guess it may be possible to call it here just after the `LoadFromAssemblyPath` call. If not, it is possible to find the `System.Console` assembly in the `AssemblyLoadContext.Assemblies` after the load into the assembly load context completes.